kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.data.DataService.xml

```diff
@@ -27,14 +27,14 @@
             default="false"
             description="Enable automatic connect of the Data Publishers on startup and after a disconnection."/>
                         
-        <AD id="connect.retry-interval"
-            name="Connect Retry-interval"
+        <AD id="backoff.max.tentatives"
+            name="Backoff max tentatives"
             type="Integer"
```

Comment:
Please change the word "tentative" with "attempt" or "try". Tentative means temporary, uncertain.